source/NVDAMagnifier/__init__.py

```diff
@@ -0,0 +1,842 @@
+import ctypes
```

Comment:
please refactor the majority of this code into files within this folder. The init file should really only have high level logic such as initialization and termination of the module.